### DISCUSSION TITLE
Toast 和 MessageBox的api

### DISCUSSION THREAD

**Discussion by smile-yyds** [kikipoulet/SukiUI#311] [Q&A]
Toast 和 MessageBox的调用

**Comment by AuroraZiling** [comment]
https://kikipoulet.github.io/SukiUI/zh/documentation/hosts/hosts.html

**Comment by smile-yyds** [comment]
引用了但是不知道为什么没有弹出来消息[image: image.png]
运行打印时出来
[image: image.png]
但是消息窗口没有显示，请教一下会是什么原因。在viewmodels   MVVM模式运行

**Reply by AuroraZiling** [reply]
我这边好像看不到图片，有没有复现仓库或者源码
